android/app/src/main/java/org/fossasia/openevent/adapters/SpeakersListAdapter.java

```diff
@@ -124,10 +127,26 @@ public void onBindViewHolder(RecyclerViewHolder holder, final int position) {
         holder.itemView.setOnClickListener(new View.OnClickListener() {
             @Override
             public void onClick(View v) {
-                String speakerName = current.getName();
-                Intent intent = new Intent(activity, SpeakerDetailsActivity.class);
-                intent.putExtra(Speaker.SPEAKER, speakerName);
-                activity.startActivity(intent);
+
+                if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP) {
+                    holder.speakerName.setTransitionName(activity.getString(R.string.activity_speaker_name_text_trans));
+                    holder.speakerImage.setTransitionName(activity.getString(R.string.activity_speaker_image_trans));
+
+                    Pair<View, String> pair1 = Pair.create((View) holder.speakerImage, holder.speakerImage.getTransitionName());
+                    Pair<View, String> pair2 = Pair.create((View) holder.speakerName, holder.speakerName.getTransitionName());
+                    ActivityOptionsCompat options = ActivityOptionsCompat.makeSceneTransitionAnimation(activity, pair1, pair2);
+                    String speakerName = current.getName();
+                    Intent intent = new Intent(activity, SpeakerDetailsActivity.class);
+                    intent.putExtra(Speaker.SPEAKER, speakerName);
+                    activity.startActivity(intent,options.toBundle());
+
```

Comment:
Please make sure that in SpeakerDetailActivity, the transition is postponed until the data is loaded.

See [this](https://www.androiddesignpatterns.com/2015/03/activity-postponed-shared-element-transitions-part3b.html)

This ensures that the UI will not look janky while transition happens